asterix/ast_data_block.ksy

```diff
@@ -0,0 +1,32 @@
+meta:
+  id: ast_data_block
+  file-extension: ast
+  endian: be
+  license: GPL-3.0-only
```

Comment:
Surveillance is evil, copyright is also evil. But since it is already under copyright, **how about AGPL here**? GPL doesn't prevent parties involved into surveillance from privately using this spec.